docs/UserStories.md

```diff
@@ -2,95 +2,102 @@
+- do everything an "anonymous user" may do
+- identify an issue with "the Web site"
+- identify an issue with "the documentation"
+- identify an issue with "the software"
+- propose new fixes for any of the above ("open a Pull Request / Merge Request")
+- participate in discussions of any of the above (proposed fixues or issues, including "review")
+- automatically run the tests of any proposed fix ("continuous integration / CI")
+- close an issue (that I created)
+
+nice-to-have:
+- understand the source lifecycle (eg. what's next when the PR is review and merged and is my ticket/issue automaticly closed?)
+- be notified of changes made on the tickets/issues I'm linked to
+- re-open any closed ticket/issue
+
 
 ## As a Tahoe-LAFS maintainer = power user
 
 I expect to be able to...
 
-1. act as a contributor (described above), in addition to the following activities
-2. be notified for new/changed wiki pages proposed by contibutors
-3. approve/publish new/changes wiki pages
-4. unpublish/delete wiki pages
-5. be notified for new tickets/issues, and possibly confidential
-6. update tickets/issue (for triage):
+- everything in the above two categories
+- be notified for new tickets/issues
+- update tickets/issue (for triage):
   - title and description
   - owner/assignee(s)
-  - type, priority, component, version, milestone, resolution
-  - other keywords/labels
-7. CRUD types, priorities, components, versions, milestones and resolutions available for tickets/issues
-8. CRUD other keywords/labels available for ticket/issues
-9. lock/close tickets/issues
+  - CRUD other metadata
+- CRUD available metadata of tickets/issues
+- close any issues
+
+nice to have:
+
+- lock any issues from further changes / discussion
+
 
 #### Questions left:
 
 - can a contributor create keywords/labels w/o priviledges?
-- how to/who should handle **confidential** tickets/issues (0-day, responsible disclosure, etc.)?
-- are code reviewers always maintainers/admins of the project?
+- how to/who should handle **confidential** tickets/issues (0-day, responsible disclosure, etc.)? [meejah: out of scope]
+- are code reviewers always maintainers/admins of the project? [meejah: currently at least write permission]
 - may issues/tickets without PR get closed with `wontfix` or `duplicate`?
 - workflow criteria for source code management: authors of approved PRs can get promoted to add commits to feature branches?
 
 ## As a Tahoe-LAFS admin = super user
 
 I expect to be able to...
 
-1. act as a maintainer (described above), in addition to the following activities
-2. add/remove maintainers
-
-## As a Tahoe-LAFS owner = root user
+- everything above
+- add / remove maintainers
+- add / remove admin
+- add / remove projects
 
-I expect to be able to...
-
-1. act as a maintainer (described above), in addition to the following activities
-2. add/remove admin
-3. add/remove projects
 
 ## As a migrator = system admin
 
+meejah: What is this role? Is it time-limited?
```

Comment:
The migrator role should be time-limited indeed. The system admin, maybe not. It will likely depend on the final target.
Anyway, this section was a mere attempt to raise concerns in a perspective that may not be well documented so far.
Let's figure it out in a next round.